dlc_security/musicrooftop.lua

```diff
@@ -6,8 +6,7 @@ MpSecurityMusicRoofTop = {
     Ipl = {
         Interior = {
             ipl = {
-                'sf_billboards', -- -592.6896, 273.1052, 116.302444
-            }
```

Comment:
You removed this I am guessing accidentally, probably needs to be added back.

Also this file is still not in fxmanifest

Reply:
Fuck my brain I first check all and checked using one file and I missed something on splitting